PlayChat/MessageViewController.swift

```diff
@@ -35,58 +35,80 @@ UITabBarControllerDelegate {
   }
 
   func tabBarController(
-      ptabBarController: UITabBarController,
-      didSelectViewController viewController: UIViewController) {
+      _ ptabBarController: UITabBarController,
+      didSelect viewController: UIViewController) {
     query?.removeAllObservers()
     query = ref.child(CHS)
       .child(ptabBarController.selectedViewController!.tabBarItem.title!)
-      .queryOrderedByChild("time").queryLimitedToLast(maxMessages)
+      .queryOrdered(byChild: "time").queryLimited(toLast: maxMessages)
     let title = String(ptabBarController.selectedViewController!
       .tabBarItem.title!)
-    let tableView : UITableView = channelViewDict[title]!
+    let tableView: UITableView = channelViewDict[title]!
     fbLog?.log(inbox, message: "Switching channel to '" + title + "'")
-    query.observeEventType(.Value, withBlock : { snapshot in
-      self.msgs = []
-      for entry in snapshot.children {
-        let msg = Message(text: String(entry.value!.objectForKey("text")!),
-          displayName: String(entry.value!.objectForKey("displayName")!))
-        msg.time = String(entry.value!.objectForKey("time")!)
-        self.msgs.append(msg)
-      }
-      tableView.reloadData()
-      if (snapshot.childrenCount > 0) {
-        let indexPath = NSIndexPath(forRow: self.msgs.count-1, inSection: 0)
-        tableView.scrollToRowAtIndexPath(indexPath,
-          atScrollPosition: UITableViewScrollPosition.Bottom, animated: false)
-      }
-    }) { (error) in
+    query.observe(
+      .value,
+      with: { snapshot in
```

Comment:
nit: This declaration can be all on one line.

Reply:
Done.